src/sentry/quotas/types.py

```diff
@@ -2,5 +2,6 @@
 
 from sentry.monitors.models import Monitor
 from sentry.uptime.models import ProjectUptimeSubscription
+from sentry.workflow_engine.models import Detector
 
-SeatObject: TypeAlias = Union[Monitor, ProjectUptimeSubscription]
+SeatObject: TypeAlias = Union[Monitor, ProjectUptimeSubscription, Detector]
```

Comment:
So is the idea to eventually remove ProjectUptimeSubscription, and then later Monitor?